source/SIL.AppBuilder.Portal/common/prisma/schema.prisma

```diff
@@ -106,6 +106,8 @@ model OrganizationMembershipInvites {
   Redeemed       Boolean       @default(false)
   InvitedById    Int
   OrganizationId Int
+  Roles          Int[]         @default([])
+  Groups         Int[]         @default([])
```

Comment:
You will need to either rerun the migration to reflect the changes, or manually edit the migration script to reflect this change.